newspack-theme/inc/color-patterns.php

```diff
@@ -303,7 +303,8 @@ function newspack_custom_colors_css() {
 				.site-footer,
 				.site-footer a,
 				.site-footer a:hover,
-				.site-footer .widget-title,
+				.site-footer .widget-title,,
+				.site-footer .widgettitle,
```

Comment:
There's an extra comma here:

```suggestion
				.site-footer .widget-title,
				.site-footer .widgettitle,
```

Reply:
Resolved here b58425d8